recipe/meta.yaml

```diff
@@ -44,12 +50,14 @@ requirements:
     - scikit-build >=0.11.0
     - setuptools >=40.1.0
     - spdlog
+    - cuquantum-python  # [linux64 and cuda_compiler_version != "None"]
```

Comment:
@wshanks I don't think Qiskit Aer depends on cuQuantum Python yet?
```suggestion
```

Reply:
I think cuquantum support was added in [this PR](https://github.com/Qiskit/qiskit-aer/pull/1854). It adds some extra dependencies in setup.py when building the qiskit-aer-gpu package for PyPI. I have not researched this much yet. I think those dependencies it lists are just libraries needed by the C++ code (but hosted on PyPI) and are separate from cuquantum-python. I don't see a reference to `cuquantum` in the Python code of qiskit-aer.